src/FrontDoor/FrontDoor/Models/PSFrontDoor.cs

```diff
@@ -28,10 +26,12 @@ public class PSFrontDoor: PSTrackedResource
 
         public PSEnforceCertificateNameCheck? EnforceCertificateNameCheck { get; set; }
 
+        public PSBackendPoolsSetting BackendPoolsSetting { get; set; }
+
         public List<PSHealthProbeSetting> HealthProbeSettings { get; set; }
 
         public List<PSLoadBalancingSetting> LoadBalancingSettings { get; set; }
-        
+
```

Comment:
You made a lot of formating improvement. Nice!

Reply:
Thanks for noticing!